programs/ssl/ssl_server2.c

```diff
@@ -2657,8 +2657,10 @@ int main( int argc, char *argv[] )
     if( opt.cache_max != -1 )
         mbedtls_ssl_cache_set_max_entries( &cache, opt.cache_max );
 
+#if defined(MBEDTLS_HAVE_TIME)
```

Comment:
Since `cache_timeout` can't be used, it's probably an idea to put the `MBEDTLS_HAVE_TIME` guard around the option handling for `cache_timeout` around line 1950.

(To be ultra-correct, we could even remove `cache_timeout` from `struct options`, but I think that's going too far - adding too much complexity for not much gain. However, allowing the user to set an option that is then ignored isn't a great user experience, hence this comment)